pkg/hintrunner/zero/zerohint_keccak.go

```diff
@@ -0,0 +1,85 @@
+package zero
+
+import (
+	"fmt"
+
+	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	VM "github.com/NethermindEth/cairo-vm-go/pkg/vm"
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/builtins"
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
+)
+
+func newCairoKeccakFinalizeHint(keccakStateSizeFeltsResOperander, blockSizeResOperander, keccakPtrEndResOperander hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "IsLeFelt",
+		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
+			//> _keccak_state_size_felts = int(ids.KECCAK_STATE_SIZE_FELTS)
+			//> _block_size = int(ids.BLOCK_SIZE)
+			//> assert 0 <= _keccak_state_size_felts < 100
+			//> assert 0 <= _block_size < 10
+			//> inp = [0] * _keccak_state_size_felts
+			//> padding = (inp + keccak_func(inp)) * _block_size
+			//> segments.write_arg(ids.keccak_ptr_end, padding)
+
+			keccakStateSizeFelts, err := hinter.ResolveAsUint64(vm, keccakStateSizeFeltsResOperander)
+			if err != nil {
+				return err
+			}
+
+			if keccakStateSizeFelts >= 100 {
+				return fmt.Errorf("assert 0 <= _keccak_state_size_felts < 100.")
+			}
+			blockSize, err := hinter.ResolveAsUint64(vm, blockSizeResOperander)
+			if err != nil {
+				return err
+			}
+			if blockSize >= 10 {
+
+				return fmt.Errorf("assert 0 <= _block_size < 10.")
+			}
+
+			var input [25]uint64
+			builtins.KeccakF1600(&input)
+			padding := make([]uint64, keccakStateSizeFelts)
+			padding = append(padding, input[:]...)
+			result := make([]uint64, 0, keccakStateSizeFelts*blockSize)
+			for i := uint64(0); i < blockSize; i++ {
+				result = append(result, padding...)
+			}
+			keccakPtrEnd, err := hinter.ResolveAsAddress(vm, keccakPtrEndResOperander)
+			if err != nil {
+				return err
+			}
+			keccakPtrEndCopy := *keccakPtrEnd
+			for i := 0; i < len(result); i++ {
+				multiplicitesSegmentWriteArgsPtr, err := keccakPtrEndCopy.AddOffset(int16(i))
```

Comment:
Will be i always smaller than than 255?

Reply:
Yeah mb, changed this to the same logic as in UsortBody (adding 1 in each iteration)